### PR TITLE
Final Changes for Release 11.1.0

### PR DESCRIPTION
**Pull request by stevenchalem**

### CODE REVIEW

**Review by Jamie-SA — COMMENTED** [review]
I didn't try to review the ontology changes, I assume they've all been reviewed previously.

I only noticed one thing that I left a comment on.

File: bundle.yaml [hunk 1]
```diff
@@ -187,9 +189,21 @@ actions:
...
-    - "gistDocumentation.md"
-    - "README.md"
-    - "TimeRelatedChanges_v11.md"
+- action: "copy"
+  message: "Copying readme to Documentation directory and patching local links."
+  source: "README.md"
+  target: "{output}/Documentation/README.md"
+  replace:
+    from: "\\(./(\\w+)\\.md\\)"
+    to: "(./\\g<1>.html)"
```
[the remark this answers is not in the document]
Comment (stevenchalem) [reply bundle.yaml:198]: Chances are @rjyounes had a reason not to remove this. We'll leave it for now.

File: docs/ReleaseNotes.md [hunk 2]
```diff
@@ -1,6 +1,38 @@
...
+
+- Added `gist:domainIncludes gist:Text` to `gist:uniqueText`.
+- Improved the clarity and accuracy of annotations on several terms, including definitions, examples, and scope notes:
+  - `gist:Building`: Issue [#482](https://github.com/semanticarts/gist/issues/482).
+  - `gist:PhysicalSubstance` and `gist:PhysicalIdentifiableItem`: Issue [#644](https://github.com/semanticarts/gist/issues/644).
+  - `gist:uniqueText`: Issue[#577](https://github.com/semanticarts/gist/issues/577).
+  - `gist:precedes`, `gist:precedesDirectly`, and `gist:followsDirectly`. Issue [#300](https://github.com/semanticarts/gist/issues/300).
+  - `gist:Template` [#408](https://github.com/semanticarts/gist/issues/408).
+- Added SHACL `PrefixDeclaration`s for prefixes used in the gist ontology. The declaration for `gist:` is in the gist core ontology, and additional prefixes are included in a separate ontology file. Issue [#684](https://github.com/semanticarts/gist/issues/684).
+- Fixed text defintion of `gist:actualStartDateTime`. [#689](https://github.com/semanticarts/gist/issues/689).
```
Comment (rjyounes) [inline docs/ReleaseNotes.md:32]: typo: definition

File: docs/ReleaseNotes.md [hunk 3]
```diff
@@ -1,6 +1,38 @@
...
+- Changed superproperty `gist:startDateTime` to `gist:actualStartDateTime` in formal definition of `gist:ContemporaryEvent`. Issue [#696](https://github.com/semanticarts/gist/issues/696).
+
+### Patch Updates
+
+- Added `gist:domainIncludes gist:Text` to `gist:uniqueText`.
+- Improved the clarity and accuracy of annotations on several terms, including definitions, examples, and scope notes:
+  - `gist:Building`: Issue [#482](https://github.com/semanticarts/gist/issues/482).
+  - `gist:PhysicalSubstance` and `gist:PhysicalIdentifiableItem`: Issue [#644](https://github.com/semanticarts/gist/issues/644).
+  - `gist:uniqueText`: Issue[#577](https://github.com/semanticarts/gist/issues/577).
+  - `gist:precedes`, `gist:precedesDirectly`, and `gist:followsDirectly`. Issue [#300](https://github.com/semanticarts/gist/issues/300).
```
Comment (rjyounes) [inline docs/ReleaseNotes.md:29]: Can you add to this line: 
`gist:precedes`, `gist:precedesDirectly`, and `gist:followsDirectly`. These predicates no longer apply to objects at the same position. Issue [#300](https://github.com/semanticarts/gist/issues/300).

**Review by rjyounes — CHANGES_REQUESTED** [review]
Just 2 more and we're good to go.

File: bundle.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (rjyounes) [reply bundle.yaml:198]: Hm, I did mean to remove it and I don't know what happened. As long as the bundler works fine, we can leave it, and I'll remove it after the release.

**Comment by rjyounes** [comment]
@stevenchalem @dylan-sa The only thing this might affect would be the display of README.html. Could you take a look at it in the release package to check that it's ok?

**Pull request merged**